daveengelberg-smangelberg/car_lot/car_lot.rb

```diff
@@ -0,0 +1,109 @@
+#1 establish inventory
+#2 Car buyer preference via user input - limit the total # of options.
+#3 Basic matching engine - it should call a bunch of other simple methods
+#4 Print cars that suit buyers... poetic license
+
+require 'pry'
+require 'pry-byebug'
+
+# Initial car creation - need to put these into a lot
+# I think this means that they should be put into an array
+nissan = { name: "Nissan", model: "Maxima", year: 1995, electric: false, color: "green", price: 4000 }
+toyota = { name: "Toyota", model: "Camry", year: 2000, electric: false, color: "gold", price: 5000 }
+tesla = { name: "Tesla", model: "Model S", year: 2015, electric: true, color: "black", price: 50000 }
+nissan_electric = { name: "Nissan", model: "Leaf", year: 2013, electric: true, color: "red", price: 6000 }
+alfa = { name: "Alfa Romeo", model: "Spyder", year: 1984, electric: false, color: "blue", price: 3500}
+
+# Creating an array of cars that I can access from any method.
+# Is this the best way to do this?
```

Comment:
You're correct that the car hashes should be put into an array, and using the push method for each car is best practice. It seems like a small deal now, but soon you'll have much more data to work with that you won't want to deal with by hand. 😜